Source/WebCore/dom/Node.h

```diff
@@ -598,18 +598,20 @@ class Node : public EventTarget, public CanMakeCheckedPtr {
     bool hasTypeFlag(TypeFlag flag) const { return OptionSet<TypeFlag>::fromRaw(m_typeBitFields).contains(flag); }
 
     enum class StateFlag : uint16_t {
-        IsLink = 1 << 0,
-        IsUserActionElement = 1 << 1,
-        IsParsingChildren = 1 << 2,
-        EverHadSmoothScroll = 1 << 3,
-        SelfOrPrecedingNodesAffectDirAuto = 1 << 4,
-        EffectiveLangKnownToMatchDocumentElement = 1 << 5,
-        IsComputedStyleInvalidFlag = 1 << 6,
-        NeedsSVGRendererUpdate = 1 << 7,
-        NeedsUpdateQueryContainerDependentStyle = 1 << 8,
+        IsLink                                      = 1 << 0,
```

Comment:
Can we not do this reformatting in this PR? I think this needs a wider discussion. The last I checked, we're not supposed to align ='s like this.